travis/publish.sh

```diff
@@ -0,0 +1,43 @@
+#!/bin/bash -e
+
+# Setup login
+openssl aes-256-cbc -K $encrypted_fb15e83f0f9f_key -iv $encrypted_fb15e83f0f9f_iv \
+    -in .dockercfg.enc -out ~/.dockercfg -d
+
+
+JDK="oraclejdk7"   # what JDK we use to build the WAR that goes in the docker image
+NAME="camptocamp/mapfish_print"
+
+function publish {
+    local version=$1
+    export DOCKER_VERSION=`echo "${version}" | sed -e 's/^release\///' | sed -e 's/\//_/g'`
+
+    echo "Deploying image to docker hub for tag ${DOCKER_VERSION}"
+    ./gradlew --console=plain createDocker
+    docker push "${NAME}:${DOCKER_VERSION}"
+}
+
+if [ "${TRAVIS_JDK_VERSION}" != "${JDK}" ]
+then
+    echo "Only publishing with ${JDK}"
+    exit 0
+fi
+
+if [ "${TRAVIS_PULL_REQUEST}" == "true" ]
```

Comment:
This Travis job was triggered by your PR:
https://travis-ci.org/mapfish/mapfish-print/jobs/94578463

And in the logs I don't see a "Not deploying image for pull requests" but:

    The push refers to a repository [docker.io/camptocamp/mapfish_print] (len: 1)

... which probably means that the image was uploaded.

Please take a look at the docs:
https://docs.travis-ci.com/user/environment-variables#Default-Environment-Variables